components/movementsensor/merged/merged.go

```diff
@@ -281,71 +281,108 @@ func (m *merged) Accuracy(ctx context.Context, extra map[string]interface{}) (ma
 	if m.ori != nil {
 		oriAcc, err := m.ori.Accuracy(ctx, extra)
 		if err != nil {
-			// replace entire map with a map that shows that it has errors
```

Comment:
Any updates to this comment here?

Reply:
good catch, deleted this line accidentally. I'll add it back